packages/atomic/src/utils/utils.ts

```diff
@@ -39,6 +39,13 @@ export function sanitize(string: string) {
   return string.replace(reg, (match) => map[match]);
 }
 
+export function filterProtocol(uri: string) {
```

Comment:
Just a general note about that file, right now it is still very small so it's okay, but those kind of `utils` files have the tendency to bloat pretty quickly because it kinda means "anything goes in this file".

So, just watch out for that !

Reply:
I'll create a separate xss-utils 😉